opencog/query/PatternMatchEngine.cc

```diff
@@ -780,8 +781,8 @@ bool PatternMatchEngine::tree_compare(const PatternTermPtr& ptm,
 	if (not match) return false;
 
 	LAZY_LOG_FINE << "depth=" << depth;
-	logmsg("> tree_compare", hp);
-	logmsg(">           to", hg);
```

Comment:
This wouldn't matter because there would be on the next line. Trust me, it doesn't make it more difficult to directly compare as the 2 atoms are still aligned. And at least there are correctly indented.